charts/all/openvino/templates/ov-notebookinstance.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: intel.com/v1alpha1
+kind: Notebook
+metadata:
+  name: v2022.3
+  namespace: redhat-ods-applications
+spec:
+  auto_update_image: false
+  git_ref: main
+  git_uri: 'https://github.com/openvinotoolkit/openvino_notebooks'
```

Comment:
If these are parameterized, it definitely makes sense to paramterize git_ref and git_uri